plenum/recorder/test/test_recorder.py

```diff
@@ -0,0 +1,181 @@
+import random
+import time
+from collections import OrderedDict
+
+from plenum.common.util import randomString
+
+try:
+    import ujson as json
+except ImportError:
+    import json
+
+import pytest
+
+from plenum.recorder.src.recorder import Recorder
+
+TestRunningTimeLimitSec = 350
+
+
+def test_add_to_recorder(recorder):
+    last_check_time = recorder.get_now_key()
+    time.sleep(1)
+    msg1, frm1 = 'm1', 'f1'
+    msg2, frm2 = 'm2', 'f2'
+    recorder.add_incoming(msg1, frm1)
+    time.sleep(3)
+    recorder.add_incoming(msg2, frm2)
+    time.sleep(2.1)
+    msg3, to1, to11 = 'm3', 't1', 't11'
+    msg4, to2 = 'm4', 't2'
+    recorder.add_outgoing(msg3, to1, to11)
+    time.sleep(.4)
```

Comment:
Do we check when multiple messages come at the same time?

Reply:
Added `test_get_list_from_recorder`